integration/integration_suite_test.go

```diff
@@ -0,0 +1,66 @@
+package integration_test
+
```

Comment:
could you organize `integration` folder into sub folders like:
```
./
├── utility
│   └── helper.go
├── test
│   ├── p2p
│   ├── metadb
│   ├── ...
 ```

Reply:
ok - I'll do in next PR